### ISSUE TITLE
Unable to use Ctrl+Enter to play 

### ISSUE THREAD

**Issue by mohammedtouheedpatelgithubcom** [CodeCafeOpenShiftGame/PodEscape#73]
**Describe the bug**
![redhat](https://user-images.githubusercontent.com/59679306/116359960-0b2dc900-a81d-11eb-95d2-c7b6db6bff70.JPG)

**To Reproduce**

**Expected behavior**

**Screenshots**

**Desktop (please complete the following information):**
 - OS: [windows 7]
 - Browser [Firefox]
 - Version [88.0 (64-bit)]

**Additional context**
I have checked my keyboard other functionalities like space and wad are working fine only ctrl+play isn't working not sure if it's a bug or not.Or maybe it's Functionality are missing.

**Comment by rluders** [comment]
All right, I actually checked this one. It seems that the canvas doesn't get the focus when we open the page. So, we probably need to add something to set it to the canvas by default when the page loads, so it can capture the keyboard events.

In order to simulate what I'm saying:

1. Open the podscape.io
2. Try to press Ctrl + Enter (it should not work)
3. Now, click in any place of the game canvas
4. Try to press Ctrl + Enter (it should now work)

We could probably add some javascript to the export template:

```javascript
document.getElementsByTagName('canvas')[0].focus()
```

And it should solve the issue.

**Comment by mohammedtouheedpatelgithubcom** [comment]
Yeah! You are absolutely correct.I too just checked now it is working.I found this luckily since my mouse wasn't working well.So I tried everything with keyboard.Thank you for your feedback @rluders .Yeah that may fix this issue.
